compiler/noirc_frontend/src/elaborator/mod.rs

```diff
@@ -1472,6 +1486,11 @@ impl<'context> Elaborator<'context> {
         }
     }
 
+    fn include_interpreter_errors(&mut self, errors: &mut [InterpreterError]) {
+        self.errors
+            .extend(errors.iter().cloned().map(InterpreterError::into_compilation_error_pair));
+    }
```

Comment:
It doesn't look like we use the interpreter after taking its errors in any of the cases we use it in - why not take them by value?